core/src/test/java/com/klaytn/caver/common/transaction/AccountUpdateTest.java

```diff
@@ -195,25 +193,25 @@ public static AccountUpdate getRoleBasedSigTx() {
                 Numeric.hexStringToByteArray("0x761e12fe11003aa4cb8fd9b44a41e5edebeb943cc366264b345d0f7e63853724")
         );
 
-        String[][] pubKeyArr = new String[][] {
+        String[][] pubKeyArr = new String[][]{
                 {
-                    "0xf7e7e03c328d39cee6201080ac2576919f904f0b8e47fcb7ea8869e7db0baf4470a0b29a1f6dd007e19a53da122d18bf6273cdddb2903ef0ad2b350b207ad67c",
-                    "0xedacd9095274f292c702514f6443f58337e7d7c8311694f31c73e86f150ecf45820929c143da861f6009784e36a6ebd99f83b1baf93fd72e820b5df3cd00883b",
-                    "0xb74fd682a6a805415e7711890bc91a283c268c78947ebf25a02a2e02625a68aa825b5213f3e9f03c34650da902a2a70915dcc1c7fe86333a7e40e638361335a4"
+                        "0xf7e7e03c328d39cee6201080ac2576919f904f0b8e47fcb7ea8869e7db0baf4470a0b29a1f6dd007e19a53da122d18bf6273cdddb2903ef0ad2b350b207ad67c",
```

Comment:
Can you check the indentation?

Reply:
@jimni1222 
Hi :) I checked with @sirano11 and we talked about "what indentations should we use?"
We decided to use below indentation setup.
```
Tab size: 4
Indent: 4
Continuation indent: 8
```
That is the right indentation based on above setting :)
Thanks for your feedback.